src/swc/aeon/io/api.py

```diff
@@ -50,12 +66,20 @@ def to_seconds(time):
         float, pandas.Series: The Harp timestamp in seconds.
     """
     if isinstance(time, pd.Series):
-        return (time - REFERENCE_EPOCH).dt.total_seconds()
+        return (pd.to_datetime(time) - REFERENCE_EPOCH).dt.total_seconds()
     else:
         return (time - REFERENCE_EPOCH).total_seconds()
 
 
-def chunk(time):
+@overload
+def chunk(time: datetime.datetime) -> pd.Timestamp: ...
+@overload
+def chunk(time: pd.DatetimeIndex) -> pd.DatetimeIndex: ...
+@overload
+def chunk(time: "pd.Series[pd.Timestamp]") -> pd.Series: ...
+def chunk(
+    time: "datetime.datetime | pd.DatetimeIndex | pd.Series[pd.Timestamp]",
```

Comment:
```suggestion
def chunk(time: pd.Series[pd.Timestamp]) -> pd.Series: ...
def chunk(
    time: datetime.datetime | pd.DatetimeIndex | pd.Series[pd.Timestamp],
```

Reply:
@lochhh good catch, I had missed that one, probably added while trying to resolve the circular dependency and wasn't coming up as unused in either `ruff check` or `pyright`.